command.go

```diff
@@ -97,7 +97,11 @@ func (cmd *Command) runLinter(args []string, opts *LinterOptions, initConfig boo
 		if err != nil {
 			return nil, fmt.Errorf("could not read stdin: %w", err)
 		}
-		return l.Lint("<stdin>", b, nil)
+		if opts.StdInFileName == "" {
+			return l.Lint("<stdin>", b, nil)
+		} else {
+			return l.Lint(opts.StdInFileName, b, nil)
+		}
```

Comment:
```suggestion
		n := "<stdin>"
		if opts.StdInFileName != "" {
			n = opts.StdInFileName
		}
		return l.Lint(n, b, nil)
```

Would you reduce repeating code?